Solutions/Nasuni/Hunting Queries/FileDeleteEvents.yaml

```diff
@@ -0,0 +1,59 @@
+id: 64a3477e-d06f-4491-86a5-6f99702e267f
+name: Nasuni File Delete Activity
+description: |
+  'This query looks for file delete audit events generated by a Nasuni Edge Appliance.'
+requiredDataConnectors:
+  - connectorId: Syslog
+    dataTypes:
+      - Syslog
+tactics:
+  - Impact
+relevantTechniques:
+  - T1485
+query: |
+  Syslog 
+  | where SyslogMessage matches regex "(nasuni.)([0-9A-Za-z]{8}-[0-9A-Za-z]{4}-[0-9A-Za-z]{4}-[0-9A-Za-z]{4}-[0-9A-Za-z]{1})"
+  | extend SyslogMessageJson = extract('{(.*?)}', 0, SyslogMessage)
+  | extend event_details = parse_json(SyslogMessageJson)
+  | extend path_parts = parse_path(tostring(event_details.path))
+  | where event_details.event_type == "AUDIT_UNLINK"
+  | project
```

Comment:
Can we summarize result?

Reply:
I don't think summarization is appropriate here. We want to see the details of the individual events, i.e., all of the files that have been deleted. Please let me know if I am missing something.